src/Npgsql/NpgsqlSlimDataSourceBuilder.cs

```diff
@@ -118,6 +119,17 @@ public NpgsqlSlimDataSourceBuilder EnableParameterLogging(bool parameterLoggingE
         return this;
     }
 
+    /// <summary>
+    /// Configure type loading options for the DataSource. Calling this again will replace
+    /// the prior action.
+    /// </summary>
+    public NpgsqlSlimDataSourceBuilder ConfigureTypeLoading(Action<NpgsqlTypeLoadingOptionsBuilder> configureAction)
```

Comment:
Maybe we should throw here if `configureAction` is null?

Reply:
You mean for #nullable disabled users?

Reply:
That or '!'. Doesn't matter that much since we'll throw NRE below anyway, but why not make it a bit nicer.